### ISSUE TITLE
Update CR3 download export file

### ISSUE THREAD

**Issue by JaceDeloney** [cityofaustin/atd-data-tech#4163]
During Sprint Review #24, the VZ team requested that we update the CR3 download export file. Here are the ideas y'all had at the time:
- Update the default sorting field
- Remove unused columns

We are still waiting on the full list of requirements from their team.

**Comment by JaceDeloney** [comment]
Attached is a file that includes:
- re-ordered fields
- fields that need to be removed in red
- fields that need to be renamed (e.g., 
[20201019 VZE Download Field List.xlsx](https://github.com/cityofaustin/atd-data-tech/files/5404572/20201019.VZE.Download.Field.List.xlsx)
death_cnt to CRIS_death_cnt)

**Comment by tylerwhitson** [comment]
Developed a complete fix, reopening this issue to fully close it with another PR
